packages/sources/css/postcss.config.js

```diff
@@ -1,7 +1,7 @@
 const postCssImport = require('postcss-import');
+const postcssColorMod = require('postcss-color-mod-function');
```

Comment:
postCssImport & postCssColorMod  or  postcssImport & postcssColorMod ?

Reply:
I don't understand your question Nicolas?

Reply:
Uppercase on C => postCssColorMod
```suggestion
const postCssColorMod = require('postcss-color-mod-function');
```
Maybe @lauthieb

Reply:
The convention seen in several documentation is `postcss` without uppercase.

Reply:
yes sorry if my comment was not clear

Reply:
in this case you can correct by `const postcssImport = require('postcss-import');`

Reply:
Yes, thanks!

Reply:
Resolved.